Cargo.toml

```diff
@@ -39,6 +39,7 @@ test-actors = { workspace = true }
 [workspace]
 members = [
     "crates/providers/*",
+    "crates/provider-sdk",
```

Comment:
Rather than doing this, could you migrate the http server provider to using this crate? Adding it here would become redundant

Reply:
I'd rather do that in a follow up

Reply:
Sure